.eslintrc

```diff
@@ -32,6 +32,7 @@
     "@typescript-eslint/no-unsafe-return": "off",
     "@typescript-eslint/prefer-optional-chain": "off",
     "@typescript-eslint/no-misused-promises": "off",
-    "@typescript-eslint/unbound-method": "off"
+    "@typescript-eslint/unbound-method": "off",
+    "@typescript-eslint/prefer-nullish-coalescing": "off"
```

Comment:
I wanted use || instead of ?? and this rule complained hahahaha that's why I turned it off, but I removed this rule right now and it's ok, IDK what happened HAHAHA